.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+**This issue tracker is a tool to address bugs in Flask-SQLAlchemy itself.
+Please use the [#pocoo IRC channel on freenode](http://flask.pocoo.org/community/irc/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/flask-sqlalchemy?sort=linked) for general
```

Comment:
Is Freenode still a good place to point people?  How about Discord?

Reply:
🤷‍♀ no opinion about Freednode, but I'm happy to add/replace with Discord. Is the best link https://discordapp.com/invite/t6rrQZH?

Reply:
I honestly don't know.  Let's see if @davidism or someone else who is more connected with Pallets than I am has a preference.  If they don't give input, then I think we could just keep what you have.